### PR TITLE
Update ruby_version requirement to allow ruby 3.0

### PR DESCRIPTION
**Pull request by kamipo**
Ruby master branch recently bumped the version number to 3.0: https://github.com/ruby/ruby/commit/21c62fb670b1646c5051a46d29081523cd782f11.

This change is required to pass bundle install with ruby-master in https://github.com/rails/rails.

https://github.com/rails/rails/blob/3d428777b05701ca7211a1be723cb1ee0e094bd9/Gemfile#L40

### CODE REVIEW

**Comment by doc75** [comment]
@kamipo , thanks for this PR.
Could you please add also in .travis.yml the version 3.0 so that tests are also passed on this supported version ?

**Comment by yahonda** [comment]
We may need to add rexml to the gemspec.

Refer https://github.com/ruby/ruby/pull/2832

**Comment by kamipo** [comment]
Yeah, I've found that in the log.
https://travis-ci.org/github/w3c-validators/w3c_validators/jobs/727920706#L274

**Comment by kamipo** [comment]
> Could you please add also in `.travis.yml` the version 3.0 so that tests are also passed on this supported version ?

Updated.

File: w3c_validators.gemspec [hunk 1]
```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'nokogiri', '~> 1.6'
   gem.add_dependency 'json', '>= 1.8'
+  gem.add_dependency 'rexml'
```
Comment (doc75) [inline w3c_validators.gemspec:19]: To keep consistency I would prefer that we add the version behind
```suggestion
  gem.add_dependency 'rexml', '~> 3.2é
```

**Review by doc75 — APPROVED** [review]

**Pull request merged**